### ISSUE TITLE
Keybase can't verify Twitter account unless Tweet is in timeline

### ISSUE THREAD

**Issue by webster** [keybase/keybase-issues#2679]
I am attempting to verify the Twitter account of an organization. I have posted the verification tweet on the organization's account, without making it appear in the timeline (i.e. I have a direct link to the tweet, but it won't appear in the timeline). However, Keybase only verifies that it's in the timeline, and doesn't accept a Tweet url. 

Not all organizations will desire to have a Keybase verification tweet publicly posted in their timeline, as it might be obscure or confusing to most of their followers. Moreover, even if one does post the verification tweet, they can't delete it or the verification later fails. However, with a posted tweet that's not in the timeline - as I have done here - it can stay online and accessible as with any other tweet, but it just won't be in the timeline.

Here's an example:

Tweet: https://twitter.com/Apple/status/771787818095874048
Not in timeline: https://twitter.com/Apple

TL;DR: please verify tweet links, not just whether it's in the timeline. Thanks!

**Comment by zQueal** [comment]
This is a really great idea. As long as the profile is unprotected and the tweet will always be public, I don't see how this will raise issue. However, new functionality will need to be added to the twitter proof verification process.
